src/Uno.UI.RemoteControl.Host/RemoteControlExtensions.cs

```diff
@@ -82,6 +83,13 @@ public static IApplicationBuilder UseRemoteControlServer(
 							}
 						}
 					}
+					catch (WebSocketException)
+					{
+						if (app.Log().IsEnabled(LogLevel.Information))
+						{
+							app.Log().LogInformation("Connection from {Remote} has been closed", context.Connection.RemoteIpAddress);
+						}
+					}
```

Comment:
Would this `catch` also address this issue we have? https://github.com/unoplatform/uno/issues/18337